datamodel/testdata.d/08_03_test_data_ovs.sql

```diff
@@ -173,7 +173,7 @@ INSERT INTO dcsa_im_v3_0.operations_event (
     'TR',
     'BUNK',
     '06aca2f6-f1d0-48f8-ba46-9a3480adfd23',
-    'POTE',
+    'PBPL',
```

Comment:
I think `BRTH` would be more correct here (on a pendantic note)

Reply:
changed